Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3.7.4-slim-buster
+FROM python:3.7-slim-bullseye
```

I had to roll this back to get the CI to run -
```suggestion
FROM python:3.7.4-slim-buster
```

---

Interesting. I was having trouble resolving packages on `buster` so I bumped to `bullseye` so it's not a big deal to me.

---

Ah, might be cos I'm testing on my arm64 mac, I'll try it with x64

---

I am too so maybe it was just a transient issue. I was also testing some other changes (using `imgkit` instead of `selenium`/`phantomjs`) so it's probably just something I misinterpreted.